conda-environment.yml

```diff
@@ -5,7 +5,8 @@ dependencies:
   - ipykernel=5.1.4
   - jupyter
   - conda-forge::mne=0.20.5
-  - conda-forge::mne-bids
+  - conda-forge::mne-bids=2.2.2
```

Comment:
Looks like mne-bids 2.2.2 doesn't exist, the latest version is 0.6. Should it be changed or I misunderstand someting?
We haven' specified mne-bids version in previous commits, I suggest 0.5 version.

Reply:
Both the server-wide environment you use and my environment use 0.4. Sheer luck the versions are the same. I'll set the version together with the correct papermill version.